### PR TITLE
Bcmohad 3577

### PR DESCRIPTION
**Pull request by monikadhillon**
Case fields, Page Layout and Profile changes

### CODE REVIEW

File: MAiD - Case Management App/.vscode/settings.json [hunk 1]
```diff
@@ -3,6 +3,5 @@
     "**/node_modules": true,
     "**/bower_components": true,
     "**/.sfdx": true
-  },
-  "salesforcedx-vscode-core.show-cli-success-msg": false
```
Comment (herman-ovodov-cgi) [inline MAiD - Case Management App/.vscode/settings.json:7]: Get latest from Dev branch

File: MAiD - Case Management App/config/project-scratch-def.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "orgName": "MAiD Dev",
```
Comment (herman-ovodov-cgi) [inline MAiD - Case Management App/config/project-scratch-def.json:2]: Get latest from Dev

File: MAiD - Case Management App/force-app/main/default/objects/Case/fields/Case_Onwer__c.field-meta.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (herman-ovodov-cgi) [inline MAiD - Case Management App/force-app/main/default/objects/Case/fields/Case_Onwer__c.field-meta.xml:1]: Replace with out of the box Case Owner field

File: MAiD - Case Management App/force-app/main/default/objects/Case/fields/Case_Type__c.field-meta.xml [hunk 4]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (herman-ovodov-cgi) [inline MAiD - Case Management App/force-app/main/default/objects/Case/fields/Case_Type__c.field-meta.xml:1]: Utilize out of the box field

File: MAiD - Case Management App/sfdx-project.json [hunk 5]
```diff
@@ -2,9 +2,7 @@
   "packageDirectories": [
     {
       "path": "force-app",
```
Comment (herman-ovodov-cgi) [inline MAiD - Case Management App/sfdx-project.json:4]: Get latest from Dev

File: MAiD - Case Management App/.vscode/settings.json (hunk 1 above)
Comment (monikadhillon) [reply MAiD - Case Management App/.vscode/settings.json:7]: Changed settings.json just as it is in dev

File: MAiD - Case Management App/config/project-scratch-def.json (hunk 2 above)
Comment (monikadhillon) [reply MAiD - Case Management App/config/project-scratch-def.json]: Scratch org def changed to be the same as dev

File: MAiD - Case Management App/sfdx-project.json (hunk 5 above)
Comment (monikadhillon) [reply MAiD - Case Management App/sfdx-project.json]: Done

File: MAiD - Case Management App/force-app/main/default/layouts/Account-Account Layout.layout-meta.xml [hunk 6]
```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (herman-ovodov-cgi) [inline MAiD - Case Management App/force-app/main/default/layouts/Account-Account Layout.layout-meta.xml:1]: Remove

File: MAiD - Case Management App/force-app/main/default/layouts/PersonAccount-Person Account Layout.layout-meta.xml [hunk 7]
```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (herman-ovodov-cgi) [inline MAiD - Case Management App/force-app/main/default/layouts/PersonAccount-Person Account Layout.layout-meta.xml:1]: Remove

File: MAiD - Case Management App/force-app/main/default/objects/Account/fields/College_ID__c.field-meta.xml [hunk 8]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
```
Comment (herman-ovodov-cgi) [inline MAiD - Case Management App/force-app/main/default/objects/Account/fields/College_ID__c.field-meta.xml:2]: Remove

File: MAiD - Case Management App/force-app/main/default/objects/Case/fields/Case_Onwer__c.field-meta.xml (hunk 3 above)
Comment (herman-ovodov-cgi) [inline MAiD - Case Management App/force-app/main/default/objects/Case/fields/Case_Onwer__c.field-meta.xml:1]: Remove

File: MAiD - Case Management App/force-app/main/default/profiles/Custom%3A Sales Profile.profile-meta.xml [hunk 9]
```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (herman-ovodov-cgi) [inline MAiD - Case Management App/force-app/main/default/profiles/Custom%3A Sales Profile.profile-meta.xml:1]: Remove

File: MAiD - Case Management App/force-app/main/default/profiles/Custom%3A Support Profile.profile-meta.xml [hunk 10]
```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (herman-ovodov-cgi) [inline MAiD - Case Management App/force-app/main/default/profiles/Custom%3A Support Profile.profile-meta.xml:1]: Remove

**Review by herman-ovodov-cgi — APPROVED** [review]
Reviewed and approved by Herman and Ralph, cleaned up by Herman

**Pull request merged**